docs/winui3.md

```diff
@@ -0,0 +1,34 @@
+---
+id: winui3
+title: WinUI 3
+---
+
+# WinUI 3 Support in RNW
+
+We are working on adding **WinUI 3** support for RNW and have made some good progress towards an alpha.

```

Comment:
Can you add some preamble on what WinUI 3 is (perhaps with links to the WinUI3, project reunion repos) and why we are doing this so there is context for people coming new?

Reply:
You could borrow from [this doc](https://microsoft.github.io/react-native-windows/docs/win10-compat) or maybe even add the content from that doc into yours if it is meaningful.